Form/Type/TemplateChoiceType.php

```diff
@@ -73,7 +73,10 @@ public function getTemplates()
      */
     public function getParent()
     {
-        return 'choice';
+        // NEXT_MAJOR: Remove ternary (when requirement of Symfony is >= 2.8)
+        return method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+            ? 'Symfony\Component\Form\Extension\Core\Type'
```

Comment:
@core23 Fix it please :)

Reply:
cf. https://github.com/sonata-project/SonataPageBundle/pull/870#pullrequestreview-67109486